kornia/geometry/transform/imgwarp.py

```diff
@@ -65,7 +54,7 @@ def warp_perspective(src: torch.Tensor, M: torch.Tensor, dsize: Tuple[int, int],
           'bilinear' | 'nearest'. Default: 'bilinear'.
         border_mode (str): padding mode for outside grid values
           'zeros' | 'border' | 'reflection'. Default: 'zeros'.
-        align_corners(bool): interpolation flag. Default: False.
+        align_corners(bool): interpolation flag. Default: True.
```

Comment:
We should probably do a proper deprecation cycle. That means we do

```python
def warp_perspective(align_corners: Optional[bool] = None):
    if align_corners is None:
        warnings.warn("align_corners will change its default value for kornia>0.6.")
        align_corners = False
```

With this only users that used the default value will see the warning.